commit 757332c643ca474dae6cfdacb5a1fce68580c82d
Author: Ye Yin <dbyin@tencent.com>
Date:   Mon Feb 4 08:54:25 2019 -0800

    fs/xfs: fix f_ffree value for statfs when project quota is set
    
    commit de7243057e7cefa923fa5f467c0f1ec24eef41d2 upsream.
    
    When project is set, we should use inode limit minus the used count
    
    Signed-off-by: Ye Yin <dbyin@tencent.com>
    Reviewed-by: Darrick J. Wong <darrick.wong@oracle.com>
    Signed-off-by: Darrick J. Wong <darrick.wong@oracle.com>
    Signed-off-by: Luis Chamberlain <mcgrof@kernel.org>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/fs/xfs/xfs_qm_bhv.c b/fs/xfs/xfs_qm_bhv.c
index 73a1d77ec187..3091e4bc04ef 100644
--- a/fs/xfs/xfs_qm_bhv.c
+++ b/fs/xfs/xfs_qm_bhv.c
@@ -40,7 +40,7 @@ xfs_fill_statvfs_from_dquot(
 		statp->f_files = limit;
 		statp->f_ffree =
 			(statp->f_files > dqp->q_res_icount) ?
-			 (statp->f_ffree - dqp->q_res_icount) : 0;
+			 (statp->f_files - dqp->q_res_icount) : 0;
 	}
 }
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1246_757332c643ca474dae6cfdacb5a1fce68580c82d.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1246_757332c643ca474dae6cfdacb5a1fce68580c82d.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
